### ISSUE TITLE
need a better explanation of rake

### ISSUE THREAD

**Issue by ipc103** [learn-co-curriculum/rake-todo#22]
The explanation of what rake is in this README only explains it in terms of Make - if you don't know what Make is, you have no clue what Rake is. 

@AnnJohn

**Comment by AnnJohn** [comment]
Agree. I think we should also minimize how much we link to outside resources within the content, unless we've already covered the information ourselves. 

Also, suggestion to split this into two or more Readmes. The first should cover what is Rake and how to install it. The other Readme(s) can cover the rest of the material since we cover a lot in this one.
